src/frontend/src/btc/services/worker.btc-wallet.services.ts

```diff
@@ -27,16 +37,25 @@ export const initBtcWalletWorker = async ({
 
 	return {
 		start: () => {
+			const isNetworkTestnet = isNetworkIdBTCTestnet(networkId);
+			const isNetworkRegtest = isNetworkIdBTCRegtest(networkId);
+			const isNetworkMainnet = isNetworkIdBTCMainnet(networkId);
+
 			worker.postMessage({
 				msg: 'startBtcWalletTimer',
 				data: {
+					btcAddress: get(
+						isNetworkTestnet
+							? btcAddressTestnet
+							: isNetworkRegtest
+								? btcAddressRegtest
+								: btcAddressMainnet
+					),
 					bitcoinNetwork: mapToSignerBitcoinNetwork({
-						network: isNetworkIdBTCTestnet(networkId)
-							? 'testnet'
-							: isNetworkIdBTCRegtest(networkId)
-								? 'regtest'
-								: 'mainnet'
-					})
+						network: isNetworkTestnet ? 'testnet' : isNetworkRegtest ? 'regtest' : 'mainnet'
+					}),
+					// only mainnet transactions can be fetched via Blockchain API
```

Comment:
How do we develop locally and test on staging?

Reply:
Unfortunately, at the moment I don't have a decent way of testing it locally - for now, I'm hardcoding my mainnet BTC address to test the transactions. For the future, we can potentially use mocked data to have some BTC txs displayed locally.

On staging though, we should be able to test it properly for mainnet addresses.

Reply:
Gotcha. Yeah then makes sense. I'm just wondering now if this is the best place for documenting that there is no transactions locally? You know what I mean? Do we have a way to make this easier to find?

Reply:
I added a new section to `HACKING`, as it seems to be the best place to put this info. I'll make sure to keep it up-to-date as we proceed with the development.